Update the documentation with up to date info and figures (#256)
shimming-toolbox/shimming-toolbox: README.rst

    :target: https://shimming-toolbox.org/en/latest/

.. |badge-twitter| image:: https://img.shields.io/twitter/follow/shimmingtoolbox.svg?style=social&label=Follow
    :alt: Twitter Follow
    :target: https://twitter.com/shimmingtoolbox

``shimming-toolbox`` is an open-source Python software package enabling
a variety of MRI shimming (magnetic field homogenization) techniques
such as
`static <https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.25587>`__
and `real-time <https://doi.org/10.1002/mrm.27089>`__ shimming for use
with standard manufacturer-supplied gradient/shim coils or with custom
"multi-coil" arrays. The toolbox provides useful set of command line tools as
well as a `fsleyes plugin <https://github.com/shimming-toolbox/fsleyes-plugin-shimming-toolbox#fsleyes-plugin-for-shimming-toolbox>`__
dedicated to make shimming more accessible and more reproducible.

Insert image of the overview

Features
________

* Inbuilt DICOM to NIFTI conversion
* Supports masking of spinal cord
* Supports the usage and creation of custom coil profiles
* Supports different shimming approaches: spherical harmonics, multi-coil, gradient shimming, dynamic, realtime

> * Built using freely available software tools: `SCT <https://spinalcordtoolbox.com/en/latest/>`__, `FSL-prelude <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FUGUE/Guide#PRELUDE_.28phase_unwrapping.29>`__, `dcm2niix <https://github.com/rordenlab/dcm2niix>`__, `dcm2bids <https://github.com/UNFmontreal/Dcm2Bids>`__

Reviewer: ```suggestion
* Inbuilt DICOM to NIfTI conversion
* Masking tools for the brain and spinal cord
* Supports the creation and usage of a variety of coil profiles: Spherical harmonics, shim-only arrays, "AC-DC" multi-coil, etc.
* Supports different shimming scenarios: dynamic (slicewise), realtime (shim modulation with respiration), gradient z-shimming, two-region shimming (e.g., fat and brain)
* Powered by freely available software tools: `FSL-prelude <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FUGUE/Guide#PRELUDE_.28phase_unwrapping.29>`__, `SCT <https://spinalcordtoolbox.com/en/latest/>`__, `dcm2niix <https://github.com/rordenlab/dcm2niix>`__, `dcm2bids <https://github.com/UNFmontreal/Dcm2Bids>`__
```

Installation
____________

> 

Reviewer 2: I see why having this heading here helps with the overview, but on the side toolbar it looks like we have two installation sections - not sure if this is confusing? What do other people think?

Reviewer: we just did a small poll and i think we prefer it the way it is now

See the :ref:`installation` page.

Usage
_____

**1. Command Line**

> The primary way to use ``Shimming Toolbox`` is through the command line. For example:

Reviewer: ```suggestion
The primary way to use ``shimming-toolbox`` is through the command line. For example:
```

.. code-block:: console

>   $ st_prepare_fieldmap -help

Reviewer: ```suggestion
  $ st_prepare_fieldmap --help
```

  Usage: st_prepare_fieldmap [OPTIONS] PHASE...

  Creates fieldmap (in Hz) from phase images. This function accommodates multiple echoes (2 or more) and phase difference. This function also accommodates 4D phase inputs, where the 4th dimension represents the time, in case multiple fieldmaps are acquired across time for the purpose of real-time shimming experiments.

  phase: Input path of phase nifti file(s), in ascending order: echo1,
  echo2, etc.

  Options:
  -mag PATH             Input path of mag nifti file
  -unwrapper [prelude]  Algorithm for unwrapping
  -output PATH          Output filename for the fieldmap, supported types : '.nii', '.nii.gz'
  -mask PATH            Input path for a mask. Used for PRELUDE
  -threshold FLOAT      Threshold for masking. Used for: PRELUDE
  -h, --help            Show this message and exit.

.. admonition:: Note

  To facilitate reproducibility, commands can be chained together in a pipeline using multiple Shimming Toolbox commands. An `example <https://github.com/shimming-toolbox/shimming-toolbox/blob/master/examples/demo_realtime_zshimming.sh>`__ script is provided.

**2. Graphical User Interface (FSLeyes)**

> ``Shimming Toolbox`` also provides a GUI via an FSLeyes plugin. See the `plugin's Github page <https://github.com/shimming-toolbox/fsleyes-plugin-shimming-toolbox#fsleyes-plugin-for-shimming-toolbox>`__ for more information.

Reviewer: ```suggestion
``shimming-toolbox`` also features a graphical user interface (GUI) via an FSLeyes plugin. See the `plugin's Github page <https://github.com/shimming-toolbox/fsleyes-plugin-shimming-toolbox#fsleyes-plugin-for-shimming-toolbox>`__ for more information.
```

Reviewer: also: we should probably redirect [there](https://shimming-toolbox-py--256.org.readthedocs.build/en/256/getting_started/installation_gui.html) instead of to the github page of the plugin

    :alt: GitHub Actions CI
    :target: https://github.com/shimming-toolbox/shimming-toolbox/actions?query=workflow%3ACI-Tests+branch%3Amaster

.. |badge-coveralls| image:: https://coveralls.io/repos/github/shimming-toolbox/shimming-toolbox/badge.svg?branch=master
    :alt: Coverage Status
    :target: https://coveralls.io/github/shimming-toolbox/shimming-toolbox?branch=master

.. |badge-doc| image:: https://readthedocs.org/projects/shimming-toolbox-py/badge/?version=latest
    :alt: Documentation Status
    :target: https://shimming-toolbox.org/en/latest/

.. |badge-twitter| image:: https://img.shields.io/twitter/follow/shimmingtoolbox.svg?style=social&label=Follow
    :alt: Twitter Follow
    :target: https://twitter.com/shimmingtoolbox

``shimming-toolbox`` is an open-source Python software package enabling
a variety of MRI shimming (magnetic field homogenization) techniques
such as
`static <https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.25587>`__
and `real-time <https://doi.org/10.1002/mrm.27089>`__ shimming for use
with standard manufacturer-supplied gradient/shim coils or with custom
"multi-coil" arrays. The toolbox provides useful set of command line tools as

> well as a `fsleyes plugin <https://github.com/shimming-toolbox/fsleyes-plugin-shimming-toolbox>`__

Reviewer: ```suggestion
well as a `fsleyes plugin <https://github.com/shimming-toolbox/fsleyes-plugin-shimming-toolbox#fsleyes-plugin-for-shimming-toolbox>`__
```
